### DISCUSSION TITLE
A recurring error message that says "Failed to upgrade team. Please try again."

### DISCUSSION THREAD

**Discussion by Doyu-Lee** [vercel/community#3328] [Help] [answered]
### Page to Investigate

https://vercel.com/teams/everland/settings/members

### Steps to Reproduce

https://github.com/orgs/vercel/discussions/3280#discussion-5413561

# Struggling with the same problem as the above link

<img width="537" alt="image" src="https://github.com/vercel/community/assets/125176463/e1617537-7352-40ba-8ea0-2dd458074559">

Dear Support Team,

I hope this email finds you well. I am writing to seek assistance with upgrading our team plan to Pro on your platform.

Unfortunately, whenever we click on the "Upgrade" button, we encounter a recurring error message that says "Failed to upgrade team. Please try again." We have made multiple attempts, but the issue persists.

Our team name is 'SeonBBang', and we are eager to proceed with the upgrade to access the additional features offered in the Pro plan.

Could you kindly investigate this matter and provide us with a solution? We have already ensured that our payment information is accurate and up-to-date.

Thank you for your attention to this matter. We look forward to your prompt response and resolution of the issue.

Best regards,

Doyu
SeonBBang

**Comment by swarnava** [comment, marked as answer]
Hi,

I just checked our logs and it seems we are trying charge $.50 to verify your card which is rejecting by your card issuing bank. Please reach out to the bank directly to make sure the transaction is authorised before trying again.

**Reply by Vlad202** [reply]
I have same error (Failed to upgrade team. Please try again.). My team is nebo-me. Can you help me please? I have $0.50 on my card.
Thank you!
